auto_dev/commands/fsm.py

```diff
@@ -106,5 +122,11 @@ def from_file(fsm_spec: str, fsm_name: str, in_type: str, output: str) -> None:
 
     fsm = INPUT_TO_FUNC[in_type](fsm_spec, label=fsm_name)
     output_to_func = {FsmType.MERMAID.value: fsm.to_mermaid, FsmType.FSM_SPEC.value: fsm.to_string}
-    output = output_to_func[output]()
-    click.echo(output)
+    result = output_to_func[output]()
+
+    if output_path:
+        with open(output_path, "w", encoding="utf-8") as f:
+            f.write(result)
+        logger.info(f"Output written to {output_path}")
```

Comment:
use pathlib, its cleaner :)

Reply:
Should have used write to file actually lmfaooo